src/analyze/analyze-security.h

```diff
@@ -4,8 +4,8 @@
 #include "sd-bus.h"
 
 typedef enum AnalyzeSecurityFlags {
-        ANALYZE_SECURITY_SHORT             = 1 << 0,
-        ANALYZE_SECURITY_ONLY_LOADED       = 1 << 1,
+        ANALYZE_SECURITY_SHORT = 1 << 0,
+        ANALYZE_SECURITY_ONLY_LOADED = 1 << 1,
         ANALYZE_SECURITY_ONLY_LONG_RUNNING = 1 << 2,
```

Comment:
table?

Reply:
This has been  fixed here: https://github.com/systemd/systemd/pull/11690/commits/aec2e75ea62ad68f62a5ea52e773979227881b36